File: 11720번 숫자의 합.cpp
#include <iostream>
using namespace std;




int sum=0;


void calc(int count, char n[100])
{
	for(int i=0; i<count; i++){
		sum += (n[i]-'0');
	}
}

int main()
{
	

	
	int count = 0;
	char str[100];
	
	scanf("%d", &count);
	
	scanf("%s", str);
	
	calc(count, str);

	cout<<sum;
	
	return 0;
}

File: 11720번 숫자의 합.py

sum=0

def calc(count, str):

	global sum


	for i in range(0, int(count)):
		sum += ord(str[i]) - ord('0')

if __name__ == '__main__':

	count = input()

	str = input()


	calc(count, str)


	print(sum)